pyanaconda/ui/tui/tuiobject.py

```diff
@@ -93,12 +93,26 @@ class TUIObject(UIScreen, common.UIObject):
     common pyanaconda.ui.common.UIObject interface"""
 
     helpFile = None
+    _pass_func = None
 
     def __init__(self, data):
         UIScreen.__init__(self)
         common.UIObject.__init__(self, data)
         self.title = u"Default title"
 
+        if TUIObject._pass_func:
+            self.password_func = TUIObject._pass_func
+
+    @classmethod
+    def set_default_password_func(cls, password_func):
+        """Set password function which will be used for all screens.
+
+        :param password_func: Function with one parameter which is text prompt returning user
+                              input.
+        :type password_func: Password function fn(text_prompt : str) -> str .
+        """
+        cls._pass_func = password_func
```

Comment:
Maybe I would use a static method. If `set_default_password_func` is called for a different class then `TUIObject`, it will have no effect.

Reply:
Actually that can be even beneficial. That way you can define subset of UIScreens which will have different password function than others.

Reply:
But `self.password_func = TUIObject._pass_func` will not set the password function to `_pass_func` of the current class. So it doesn't make sense to support `set_default_password_func`  for different classes if it actually doesn't change the default password function at all.

Reply:
Ohh, true I've missed that. But I think better solution would be to change that assignment to be work in child methods.
If I'm not mistaken abstract that to the classmethod and call this method through `self` should be enough to have this working as expected.

Reply:
Ok, that should work. Maybe also rename `_pass_func` to `_default_password_func`, so it is clear what the attribute does.

Reply:
okay